### PR TITLE
Bug 1225615 - [Messages][Desktop] Add navigator.mozSetMessageHandler desktop-only mock. r=julien

### PR DESCRIPTION
**Pull request by azasypkin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=95956c15202939545a5b1cf4af4923169fc1bba3

File: apps/sms/index.html [hunk 1]
```diff
@@ -50,6 +50,7 @@
     <script defer src="mocks/navigator_moz_icc_manager.js"></script>
     <!-- Firefox Desktop shims -->
     <script defer src="desktop-mock/navigator_moz_has_pending_message.js"></script>
+    <script defer src="desktop-mock/navigator_moz_set_message_handler.js"></script>
```
Comment (julienw) [inline apps/sms/index.html]: why not, but maybe we could put it in moz_has_pending_message.js (and rename it to moz_system_messages.js) ?

Comment (azasypkin) [inline apps/sms/index.html]: Yep, sounds like a good idea - merging them....

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=18e3df0269b769182a9e4fe8b77e0d0aa276ed89

**Pull request merged**